compose/compose-file.md

```diff
@@ -1143,8 +1143,14 @@ string. In the example above, if `EXTERNAL_PORT` is not set, the value for the
 port mapping is `:5000` (which is of course an invalid port mapping, and will
 result in an error when attempting to create the container).
 
-Both `$VARIABLE` and `${VARIABLE}` syntax are supported. Extended shell-style
-features, such as `${VARIABLE-default}` and `${VARIABLE/foo/bar}`, are not
+In the case of environment variables that are not set, it is also possible to
+define default values like so: `${VARIABLE:-default}`. In this case, Compose
```

Comment:
Consider rewriting this a bit:

...define default values using one of the following syntax variants.

- To provide a default value which will be used if `VARIABLE` is unset or empty:

   ```
   ${VARIABLE:-default}
   ```

- To provide a default value if `VARIABLE` is empty:
   ```bash
   ${VARIABLE-default}
   ```

Both...